openml/datasets/functions.py

```diff
@@ -169,29 +171,32 @@ def list_datasets(offset=None, size=None, status=None, **kwargs):
         If qualities are calculated for the dataset, some of
         these are also returned.
     """
-    api_call = "data/list"
-    if offset is not None:
-        api_call += "/offset/%d" % int(offset)
 
-    if size is not None:
-        api_call += "/limit/%d" % int(size)
+    return openml.utils.list_all(_list_datasets, offset=offset, size=size, status=status, **kwargs)
```

Comment:
This looks to be >80 lines. Could you please try to stick to PEP8 as much as possible?

Reply:
True, I will keep the line length under 80. Regarding the arguments documentation, I kept the arguments as they were since we did not discuss any changes to them. In this case they can be included in the **kwargs as they are not positional and can all be set in one go. However, in other functions when we have lists, I would say we keep the list arguments separate from the primitives.

Reply:
Hm, given the structure of the REST API and the fact that the usage of the method right now is really unclear, it would be good if revert function signature. Also, I think calling it `**filters` instead of `**kwargs` will make the whole thing more similar to the REST API.